images/win/scripts/Installers/Install-SQLPowerShellTools.ps1

```diff
@@ -19,3 +19,10 @@ Install-Binary -Url $SharedManagementObjectsUrl -Name $SharedManagementObjectsNa
 $PowerShellToolsName = "PowerShellTools.msi"
 $PowerShellToolsUrl = "${BaseUrl}/${PowerShellToolsName}"
 Install-Binary -Url $PowerShellToolsUrl -Name $PowerShellToolsName
+
+# install sqlserver PS module
+Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+Install-Module -Name SqlServer -AllowClobber
```

Comment:
Already installed as part of - https://github.com/actions/virtual-environments/blob/21165eb5784546cfe909ccffe688b2f1ebb7b30a/images/win/toolsets/toolset-2019.json#L87